website/src/components/Showcase/Showcase.js

```diff
@@ -10,19 +10,18 @@ function Showcase() {
 }
 
 const _makeShuffledCardList = () => {
-  return cardList.sort(() => 0.5 - Math.random());
+  const alwaysOnTop = cardList.filter(card => card.shouldStayOnTop);
+  const shuffled = cardList.filter(card => !card.shouldStayOnTop).sort(() => Math.random() - 0.5);
+  return alwaysOnTop.concat(shuffled);
```

Comment:
[Lodash partition[(https://lodash.com/docs/4.17.15#partition) 🤔

Reply:
I thought about that but didn't care enough 😄

Reply:
anyway https://github.com/wix/Detox/pull/3696